connect/runtime/src/test/java/org/apache/kafka/connect/integration/BlockingConnectorTest.java

```diff
@@ -406,6 +411,7 @@ public Block(String block) {
                     blockLatch.countDown();
                 }
                 blockLatch = new CountDownLatch(1);
+                clearLatch = new CountDownLatch(1);
```

Comment:
Thanks @gharris1727 - I had a question  regarding lines 410 to 412.

When is it used and do we need to do the same for clearLatch ?